djangosaml2/views.py

```diff
@@ -250,14 +250,18 @@ def get(self, request, *args, **kwargs):
 
         # SSO options
         sign_requests = getattr(conf, '_sp_authn_requests_signed', False)
-        if sign_requests:
-            sso_kwargs["sigalg"] = getattr(conf, '_sp_signing_algorithm',
-                                           saml2.xmldsig.SIG_RSA_SHA256
-            )
-            sso_kwargs["digest_alg"] = getattr(conf,
-                                               '_sp_digest_algorithm',
-                                               saml2.xmldsig.DIGEST_SHA256
-            )
+        csc = getattr(settings, 'SAML_CONFIG', {}).get(
```

Comment:
This code is definitevely changed, please update your branch
https://github.com/IdentityPython/djangosaml2/blob/8c7a5e92d48a473681575e8f7b6a176df019ec0b/djangosaml2/views.py#L251